recipe/recipe.yaml

```diff
@@ -0,0 +1,101 @@
+schema_version: 1
+
+context:
+  name: katago
+  version: 1.15.3
+  cuda: ${{ "true" if cuda_compiler_version != "None" else "false" }}
+  cuda_build_string: cuda_${{ cuda_compiler_version | version_to_buildstring }}
+  string_prefix: ${{ cuda_build_string if cuda == "true" else "cpu_" }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+# NOTE: Disabling since it fails on Windows and replacing by bit cloning during the build for the moment.
+# source:
+#   # use git_url, because katago needs a git repo to compile BUILD_DISTRIBUTED support;
+#   # this cannot (or rather: should not) be patch out, or even patched at all (-> dirty tag),
+#   # compare the exceedingly clear instructions/pleas here:
+#   # https://github.com/lightvector/KataGo/blob/master/Compiling.md#building-for-distributed
```

Comment:
why can't you use the arhcive as part o the tag:
https://github.com/lightvector/KataGo/releases/tag/v1.15.3

that helps the bots too

Reply:
It's because katago requires a git clone at build time. You can check the cmake patch I made at https://github.com/conda-forge/katago-feedstock/pull/13